lms/djangoapps/teams/static/teams/js/spec/edit_team_spec.js

```diff
@@ -105,6 +106,7 @@ define([
             AjaxHelpers.respondWithJson(requests, _.extend(_.extend({}, teamsData), { id: '123'}));
 
             expect(teamEditView.$('.create-team.wrapper-msg .copy').text().trim().length).toBe(0);
+            expect(Backbone.trigger.calls[0].args[0]).toBe('teamModel:add');
```

Comment:
I don't think that you need this test. This is more of an integration test which you have covered with the Bok Choy test. The only Jasmine test I would want to see is that the topic view refreshes itself correctly when a team is added to a topic it is showing.